src/legacy/core_plugins/vis_default_editor/public/components/controls/components/number_list/utils.ts

```diff
@@ -76,6 +76,16 @@ function validateOrder(list: Array<number | undefined>) {
   return result;
 }
 
+function validateUniqueness(list: Array<number | undefined>): number[] {
```

Comment:
`validate` to me indicates a `boolean` return value. Might be better to name this `getDuplicateIndices`.

Reply:
fixed